Makefile.am

```diff
@@ -16,16 +16,19 @@ libopx_nas_ndi_la_SOURCES= src/hal_shell.c  src/nas_ndi_init.c \
            src/nas_ndi_route.c src/nas_ndi_utils.cpp \
            src/nas_ndi_fc_stat.cpp  src/nas_ndi_packet.c src/nas_ndi_qos_policer.cpp \
            src/nas_ndi_router_interface.c \
-           src/nas_ndi_hash.c src/nas_ndi_plat_stat.c  src/nas_ndi_qos_port.cpp \
+           src/nas_ndi_hash.c src/nas_ndi_qos_port.cpp \
            src/nas_ndi_sflow.cpp \
 	   src/nas_ndi_obj_cache.cpp \
 	   src/nas_ndi_qos_port_pool.cpp \
 	   src/nas_ndi_bridge.cpp \
 	   src/nas_ndi_l2mc.cpp \
 	   src/nas_ndi_mcast.cpp \
            src/nas_ndi_qos_wred_ecn.cpp \
-           src/nas_ndi_vlan.cpp 
-
+           src/nas_ndi_vlan.cpp \
+           src/nas_ndi_tunnel.cpp \
+           src/nas_ndi_tunnel_map.cpp \
+           src/nas_ndi_tunnel_obj.cpp
+#src/nas_ndi_plat_stat.c
```

Comment:
Why comment this line out instead of deleting it?

Reply:
Also, has the source file been deleted from the repo? I don't see that in this commit

Reply:
I'll delete it. This file shouldn't be here and should of been gone a long time ago (not part of this sync). I left this commented out as a placeholder so I can do further research on why this file remained here. I think it's safe to delete. I can always revert down the road.

Reply:
Please revert the change for nas_ndi_plat_stat.c . We need this. We got rid of nas-platform-* repos and brought this file here.